engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/query/NadelFieldToService.kt

```diff
@@ -43,7 +43,7 @@ internal class NadelFieldToService(
     }
 
     private fun getService(overallField: ExecutableNormalizedField): Service {
```

Comment:
I think a better solution might be to just hardcode the `Service` here. That way it doesn't affect the blueprint.

We do something like

```kotlin
fun getService(field: ExecutableNormalizedField): Service {
  // Guard
  if (field.name.startsWith("__")) {
    return introspectionService
  }

  // Normal code here
  return …
}
```